_test/check_exercises.nim

```diff
@@ -63,9 +63,10 @@ Options:
   -t, --tmp       Set output path to a temporary location (e.g. in `/tmp`)"""
   quit(0)
 
-let
-  appDir = getAppDir()
-  exercisesDir = appDir / ".." / "exercises"
+const
+  thisDir = currentSourcePath().parentDir()
+  exercisesDir = thisDir.parentDir() / "exercises"
```

Comment:
For this you should be able to use [`/../`](https://nim-lang.org/docs/os.html#%2F..%2F%2Cstring%2Cstring) if that helps at all.